components/renku_data_services/resource_pool_adapters/adapter.py

```diff
@@ -129,7 +129,18 @@ async def get_resource_pools(
     ) -> List[models.ResourcePool]:
         """Get resource pools from database."""
         async with self.session_maker() as session:
-            stmt = select(schemas.ResourcePoolORM).options(selectinload(schemas.ResourcePoolORM.classes))
+            stmt = select(schemas.ResourcePoolORM).options(
+                selectinload(
+                    schemas.ResourcePoolORM.classes,
+                ).selectinload(
```

Comment:
Should we just define that affinities/tolerations are loaded by default on the ORM class level. if we always load it anyways it'd clean up the code in this file a bit.

I.e. `... = relationship(..., lazy="selectin")`

Reply:
Yes this cleaned up a lot of crap. Thank you!